plaid/identity.go

```diff
@@ -19,10 +19,11 @@ type Address struct {
 }
 
 type AddressData struct {
-	City   string `json:"city"`
-	State  string `json:"state"`
-	Street string `json:"street"`
-	Zip    string `json:"zip"`
+	City       string `json:"city"`
```

Comment:
we need to add `AssetsAddressData` - the change does not apply to `AssetReport` json.

Reply:
the current assets identity and transactions data is not under the assets.go. should we add it in?

Reply:
Good catch! Lets keep it as a separate change from this PR just to keep the scope isolated.